### PR TITLE
11.0 [FIX] Added parent menu Stock for stock_scanner_XXX modules

### PR DESCRIPTION
**Pull request by misern2**
The same fix as #148

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Do you know why Travis is red?

**Comment by misern2** [comment]
@pedrobaeza travis fixed.

File: stock_scanner/models/scanner_hardware.py [hunk 1]
```diff
@@ -737,12 +737,18 @@ def _scenario_list(self, parent_id=False):
         """
 
         scanner_scenario_obj = self.env['scanner.scenario']
+
```
Comment (pedrobaeza) [inline stock_scanner/models/scanner_hardware.py]: Why adding these useless empty lines?

**Comment by pedrobaeza** [comment]
@sylvain-garancher do you agree with this change?

**Comment by pedrobaeza** [comment]
@misern2 remember to change module version number

**Review by Garamotte — APPROVED** [review]
Code review only.
LGTM, without the useless empty lines.

**Comment by misern2** [comment]
@pedrobaeza Travis is red again because of transifex. Any clue how to solve that ?
thanks.

**Comment by sbidoul** [comment]
Travis redness looks like a transient error. I restarted the job. Let's see.

**Comment by pedrobaeza** [comment]
I bump the module version number on a later commit.

**Pull request merged**